script/release/10_uploadBinariesToFilemgmt.sh

```diff
@@ -1,156 +0,0 @@
-scp -i $drools_ssh $uploadDir/business-central-$kieVersion-*.war $droolsHtdocs/$kieVersion
-scp -i $drools_ssh $uploadDir/kie-server-distribution-$kieVersion.zip $droolsHtdocs/$kieVersion
-# updatesite
-scp -r -i $drools_ssh $uploadDir/updatesite/* $droolsHtdocs/$kieVersion/org.drools.updatesite
-# docs
-scp -r -i $drools_ssh $uploadDir/drools-docs/* $droolsDocs/$kieVersion/drools-docs
-scp -r -i $drools_ssh $uploadDir/kie-api-javadoc/* $droolsDocs/$kieVersion/kie-api-javadoc
-
-
-# *** copies jbpm binaries and docs to filemgmt.jboss.org **********************************
-
-# bins
-scp -i $jbpm_ssh $uploadDir/jbpm-$kieVersion-bin.zip $jbpmHtdocs/$kieVersion
-scp -i $jbpm_ssh $uploadDir/jbpm-$kieVersion-examples.zip $jbpmHtdocs/$kieVersion
-scp -i $jbpm_ssh $uploadDir/jbpm-server-$kieVersion-dist.zip $jbpmHtdocs/$kieVersion
-# uploads jbpm -installers them to filemgt.jboss.org
-uploadInstaller(){
-        # upload installers to filemgmt.jboss.org
-        scp -i $jbpm_ssh jbpm-installer-$kieVersion.zip $jbpmHtdocs/$kieVersion
-}
-
-uploadAllInstaller(){
-        # upload installers to filemgmt.jboss.org
-        scp -i $jbpm_ssh jbpm-installer-$kieVersion.zip $jbpmHtdocs/$kieVersion
-        # upload installers to filemgmt.jboss.org
-        scp -i $jbpm_ssh jbpm-installer-full-$kieVersion.zip $jbpmHtdocs/$kieVersion
-}
-
-if [[ $kieVersion == *"Final"* ]] ;then
-        uploadAllInstaller
-else
-        uploadInstaller
-fi
-# updatesite
-scp -r -i $jbpm_ssh $uploadDir/updatesite/* $jbpmHtdocs/$kieVersion/updatesite
-# docs
-scp -r -i $jbpm_ssh $uploadDir/jbpm-docs/* $jbpmDocs/$kieVersion/jbpm-docs
-scp -r -i $jbpm_ssh $uploadDir/service-repository/* $jbpmHtdocs/$kieVersion/service-repository
-
-
-# *** copies optaplanner binaries and docs to filemgmt.jboss.org ***************************
-
-# bins
-scp -i $optaplanner_ssh $uploadDir/optaplanner-distribution-$kieVersion.zip $optaplannerHtdocs/$kieVersion
-scp -i $optaplanner_ssh $uploadDir/optaweb-employee-rostering-distribution-$kieVersion.zip $optaplannerHtdocs/$kieVersion
-scp -i $optaplanner_ssh $uploadDir/optaweb-vehicle-routing-distribution-$kieVersion.zip $optaplannerHtdocs/$kieVersion
-# docs
-scp -r -i $optaplanner_ssh $uploadDir/optaplanner-docs/* $optaplannerDocs/$kieVersion/optaplanner-docs
-scp -r -i $optaplanner_ssh $uploadDir/optaweb-employee-rostering-docs/* $optaplannerDocs/$kieVersion/optaweb-employee-rostering-docs
-scp -r -i $optaplanner_ssh $uploadDir/optaweb-vehicle-routing-docs/* $optaplannerDocs/$kieVersion/optaweb-vehicle-routing-docs
-scp -r -i $optaplanner_ssh $uploadDir/optaplanner-javadoc/* $optaplannerDocs/$kieVersion/optaplanner-javadoc
-scp -r -i $optaplanner_ssh $uploadDir/optaplanner-wb-es-docs/* $optaplannerDocs/$kieVersion/optaplanner-wb-es-docs
-
-
-# clean upload files
-rm upload_*
-
-# runs create_filemgmt_links.sh
-sh droolsjbpm-build-bootstrap/script/release/create_filemgmt_links.sh $kieVersion
```

Comment:
Is this intentional? Is it safe to remove this file completely?

Reply:
yes - becasue this is an old script not used any more